src/main/java/com/sender/email/Email.java

```diff
@@ -0,0 +1,161 @@
+package com.sender.email;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import java.io.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+@Component
+public class Email implements EmailDataAccess, Serializable {
```

Comment:
This class doesn't look like a good candidate to be component. It's lifecycle is short and the main responsibility is to encapsulate data. Thus it's fine to create new instances by hand. However, if you want to prefill some fields based on environment configuration you can make this component **prototype**, to get fresh instance for each new email